calebwagner/python-server
# creating database tables in SQL files

#### Changes Made
1. Added file `kennel.sql` and `kennel.db` to `python-server` directory.
1. Modified file `kennel.sql` and `kennel.db` to include new database for location, customer, animal, and employee lists.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-ch8
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > check in modified files for above changes
4. View code file.
    > Confirm file modifications are present as indicated above.
